assets/js/base/components/checkbox-list/index.tsx

```diff
@@ -132,15 +133,11 @@ const CheckboxList = ( {
 								! showExpanded &&
 								index >= limit && { hidden: true } ) }
 						>
-							<input
-								type="checkbox"
-								id={ option.value }
-								value={ option.value }
-								onChange={ ( event ) => {
-									onChange( event.target.value );
-								} }
-								checked={ checked.includes( option.value ) }
-								disabled={ isDisabled }
+							<FilterElementCheckbox
```

Comment:
I know the styles are different (I asked about that here: pdnLyh-1SH-p2#comment-1245), but I wonder if we could reuse the [`CheckboxControlProps` component](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/packages/checkout/components/checkbox-control/index.tsx) anyway. It might require adding a prop to control the different styles, but that way we would need to maintain one less component. Do you think that would work, @tjcafferkey?

Reply:
Thanks @Aljullu I'll give this a look into as well!